Gemfile.in

```diff
@@ -0,0 +1,10 @@
+source 'http://rubygems.org'
+
+gem 'json'
+gem 'sinatra'
+gem 'rack', '>= 1.1'
+
+Dir["#{File.dirname(__FILE__)}/bundler.d/*.rb"].each do |bundle|
+ # puts "adding custom gem file #{bundle}"
+  self.instance_eval(Bundler.read_file(bundle))
+end
```

Comment:
This is specific to bundler_ext, we don't have it in git, but create this file during packaging. Please remove it from the PR.